_layouts/default.html

```diff
@@ -1,17 +1,380 @@
+          <div class="flex download-buttons">
+            <div class="flex flex--vertical m-r">
+              <label>Production</label>
+              <a class="flex grow" id="cylc7-button">
+                <button class="button flex grow">
+                    <img src={{'/assets/icon-download.svg'}} />
+                    <span id="cylc7-latest-release-name"></span>
+                </button>
+              </a>
+              <label id="cylc7-latest-release-pubdate-text"></label>
+            </div>
+            <div class="flex flex--vertical m-l">
+                <label>Preview</label>
+                <a class="flex grow" id="cylc8-button">
+                  <button class="button flex grow">
+                    <img src={{'/assets/icon-download.svg'}} />
+                    <span id="cylc8-latest-release-name"></span>
+                  </button>
+                </a>
+                <label id="cylc8-latest-release-pubdate-text"></label>
+              </div>
+          </div>
+        </div>
+        <div class="flex flex--vertical release jc-space-around">
+            <div class="flex flex--vertical">
+              <h3 class="flex jc-center">Next Release:&nbsp;
+                <a class="page-link" id="milestone-url" target="_blank"><span id="milestone-version"></span> (<span id="progress-info"></span>)</a>
+              </h3>
+              <div class="flex">
+                <div class="progressbar" id="milestone-container">
+                  <div class="progress" id="progress"></div>
+                  <div class="flex label jc-space-between">
+                    <span>0%</span>
+                    <span>100%</span>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="flex flex--vertical">
+              <div class="flex ai-center jc-space-between">
+                <div class="flex">
+                  <h4 class="flex"><b>Previous Release</b></h4>
+                </div>
+                <a href="https://github.com/cylc/cylc.github.io" target="_blank">
+                  <button class="button transparent flex repel">
+                    <img src={{'/assets/icon-github.svg'}} alt="Clone Github repo"/>
+                    Clone CYLC
+                  </button>
+                </a>
+              </div>
+              <div class="flex flex--vertical">
+                  <a target="_blank" id='p0'></a>
+                  <a target="_blank" id='p1'></a>
+                  <a target="_blank" id='p2'></a>
+              </div>
```

Comment:
We are getting rid of the list of 3 most recent previous releases too (see current non-responsive site) - so you can remove this and any associated code.